app/posts/apply-for-teacher-training/2020-01-16-question-protocol-for-pilot.md

```diff
@@ -0,0 +1,1722 @@
+  }
+}) }}
+
+## Referees (two required)
+
+The reference process is a point of differentiation with UCAS, and requires more research to fully understand candidate and provider needs.
+
+Candidates can update their answers before submitting, but not after. If a referee declines to give a reference, or fails to respond, they can submit details of a new referee.
+
+{{ appProtocol({
+  title: "✘ Reference type",
+  items: {
+    question: "What kind of reference is this?",
+    required: "Yes",
+    api: {
+      param: "N/A"
+    },
+    why: [{
+      who: ["service"],
+      what: "uses this to determine which questions we ask a referee"
+    }],
+    what: ["Answer not submitted as part of application"]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "Referee - name",
+  items: {
+    question: "Full name",
+    required: "Yes",
+    api: {
+      param: "referees[].name",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#reference-object"
+    },
+    why: [{
+      who: ["service"],
+      what: "uses this to address referee in email correspondence"
+    }],
+    what: ["Appears in support interface. If the referee provides a reference, appears in the provider-facing interface and retrievable via API"],
+    accuracy: ["Not checked. We could offer referees the ability to correct this information, but that might raise concerns about the true identity of the referee"]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "Referee - email address",
+  items: {
+    question: "Email address",
+    required: "Yes",
+    api: {
+      param: "referees[].email",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#reference-object"
+    },
+    why: [{
+      who: ["service"],
+      what: "uses this to contact a referee and request a reference"
+    }, {
+      who: ["provider"],
+      what: "use this to validate that a reference is from an official work source (with the domain name used as a proxy)"
+    }],
+    what: ["Appears in support interface. If the referee provides a reference, appears in the provider-facing interface and retrievable via API"],
+    accuracy: [
+      "Address format is validated",
+      "If email to the nominated referee bounces, we can request correct details from the candidate"
+    ],
+    notes: ["Should providers be assessing based on this information? We don’t surface to candidates that they need to submit professional email addresses. How do we accommodate candidates (career changers, people returning to the workplace, etc.) who may struggle to provide such information?"]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "Referee - relation
```

Comment:
Referee might corroborate (thought we don't currently show to them)

Reply:
Updated.